app/views/users/update.jbuilder

```diff
@@ -0,0 +1,12 @@
+json.id @user.id
```

Comment:
is it the same as `app/views/users/show.jbuilder` ? if so, you can tell your controller#update method to use that same template with `render :show`